pause/pause.c

```diff
@@ -0,0 +1,35 @@
+//
```

Comment:
Can't be done in the build environment since is an isolated VM. What I can do is to provide de source file along the packages source files in OBS the same way we provide the golang tarball.